database/migrations/2023_08_25_080402_create_products_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->text('description');
+            $table->string('manufacturer');
+            $table->text('release_date');
+            $table->integer('price');
```

Comment:
add 
`$table->enum('price', Currencies::class)->default(Currencies::BYN->value);`

add to `app\Enums`

`enum Currencies {}` with implementation

Reply:
add cast to model

```
protected casts = [
    'currency' => Currencies::class,
];
```